src/control/cmd/daos/property.go

```diff
@@ -674,7 +674,7 @@ func createPropSlice(props *C.daos_prop_t, numProps int) propSlice {
 func allocProps(numProps int) (props *C.daos_prop_t, entries propSlice, err error) {
 	props = C.daos_prop_alloc(C.uint(numProps))
 	if props == nil {
-		return nil, nil, errors.New("failed to allocate properties list")
+		return nil, nil, errors.New("failed to allocate properties list: DER_NOMEM(-1009): Out of memory")
```

Comment:
I couldn't work out what to do here, it seems most go code is automatically adding the above based on the RC but in order to make the test pass I just added the raw text here.  Not ideal so if there's a way to construct this automatically then it would be better.

Reply:
I think you could do this:
```go
        return nil, nil, errors.Wrap(drpc.DaosNoMemory, "failed to allocate properties list")
```

The `drpc.DaosStatus` type implements the error interface and will print a nice stringified version of the error.